src/main/java/org/apache/datasketches/kll/KllDoublesSketch.java

```diff
@@ -762,65 +447,51 @@ public void merge(final KllDoublesSketch other) {
     if (Double.isNaN(maxValue_) || other.maxValue_ > maxValue_) { maxValue_ = other.maxValue_; }
     n_ = finalN;
 
-    assertCorrectTotalWeight();
+    assert KllHelper.sumTheSampleWeights(numLevels_, levels_) == n_;
```

Comment:
I tried to find this as an invariant that was documented in the base class, but it does not appear to be listed:
```
Invariants:
 
1) After a compaction, or an update, or a merge, all levels are sorted except for level zero.
2) After a compaction, (sum of capacities) - (sum of items) >= 1, so there is room for least 1 more item in level zero.
3) There are no gaps except at the bottom, so if levels_[0] = 0, the sketch is exactly filled to capacity and must be compacted.

```

Reply:
It is an invariant as well as another invariant that I added to that text in the BaseKllSketch class.